operators/src/upgrade/helm/client.rs

```diff
@@ -0,0 +1,249 @@
+use serde::Deserialize;
+use std::{
+    path::PathBuf,
+    process::{Command, Output},
+};
+
+use crate::upgrade::common::error::Error;
+
+/// Helm arguments that are required to run helm commands.
+#[derive(Debug, Clone, Default)]
+struct HelmArgs {
+    name: Option<String>,
+    opts: Vec<(String, String)>,
+    namespace: Option<String>,
+    values: Vec<PathBuf>,
+}
+
+impl HelmArgs {
+    /// Set a name.
+    fn set_name(&mut self, name: String) {
```

Comment:
nit: Looks like this is used to build the `HelmClient`. Consider using the builder pattern instead of set_ functions.